### PR TITLE
feat(component-labels): add 2 new optional properties 

### PR DESCRIPTION
**Pull request by Villanuevand**
```
[cardLabels]="{}"
 [formLabels]="{}"
```

Are 2 new properties to add the capability to modify: 
- Card Labels
- Form Labels

If those 2 options are not specified we are providing values by default.

### CODE REVIEW

File: projects/ngx-interactive-paycard-demo/src/app/app.component.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import { Component } from '@angular/core';
+import {CardLabel, FormLabel} from 'ngx-interactive-paycard-lib';
```
Comment (milantenk) [inline projects/ngx-interactive-paycard-demo/src/app/app.component.ts:2]: Could you please add a whitespace before `CardLabel` and after `Formlabel` to follow the conventions. (If you use VS Code, `ALT+SHIFT+F` does it with the default settings.)

File: projects/ngx-interactive-paycard-lib/src/lib/card/card.component.ts [hunk 2]
```diff
@@ -3,6 +3,8 @@ import { trigger, transition, state, animate, style } from '@angular/animations'
 
 import { CardModel } from '../shared/card-model';
 import { FocusedElement } from '../shared/focused-element';
+import {CardLabel} from '../shared';
+
```
Comment (milantenk) [inline projects/ngx-interactive-paycard-lib/src/lib/card/card.component.ts:7]: Could you please add a whitespace before and after `CardLabel` to follow the conventions. And please remove the unnecessary line.

File: projects/ngx-interactive-paycard-lib/src/lib/shared/default-component-labels.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  FORM_EXPIRATION_MONTH = 'Month',
+  FORM_EXPIRATION_YEAR = 'Year',
+  FORM_CVV = 'CVV',
+  FORM_SUBMIT_BUTTON = 'Submit',
+  CARD_EXPIRES = 'Expires',
+  CARD_HOLDER_NAME = 'Card Holder',
+  CARD_FULL_NAME = 'Full Name',
+  CARD_EXPIRATION_YEAR_FORMAT = 'YY',
+  CARD_EXPIRATION_MONTH_FORMAT = 'MM'
+
```
Comment (milantenk) [inline projects/ngx-interactive-paycard-lib/src/lib/shared/default-component-labels.ts:14]: Could you please remove this empty line?

**Review by milantenk — CHANGES_REQUESTED** [review]
Good job, the PR looks really great! 
I had only some minor comments regarding code conventions and the READMEs. Could you please check them?

**Comment by Villanuevand** [comment]
@milantenk 
Sure I'll do...  Im glad you like it!

**Pull request merged**